README.md

```diff
@@ -11,7 +11,60 @@ The backend for a generic mobile-friendly quiz engine created using FastAPI and
 
 ### Local DB Setup
 
-- The following steps are for a Mac.
+#### Linux Systems
+The following steps are for a Linux system.
+- To run the backend locally, you would need to setup a local instance of mongodb. The offical instructions [here](https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-ubuntu/) are good and simple enough to follow. Those steps are also listed down below.
+
+  - Import the public key used by the package management system.
+  Issue the following command to import the MongoDB public GPG Key from [here](https://pgp.mongodb.com/server-6.0.asc):
+    ```
+    curl -fsSL https://pgp.mongodb.com/server-6.0.asc | \
+    sudo gpg -o /usr/share/keyrings/mongodb-server-6.0.gpg \
+    --dearmor
+    ```
+
+  - From a terminal, install gnupg if it is not already available:
+    ```
+    sudo apt-get install gnupg
+    ```
+  - Create a list file for MongoDB.
+    Create the list file `/etc/apt/sources.list.d/mongodb-org-6.0.list` for your version of Ubuntu. To check the Ubuntu version the host is running, open a terminal or shell on the host and execute
+    ```
+    lsb_release -dc
+    ```
+
+    According to the version, run the respective code on the terminal.
+
+      - Ubuntu 22.04 (Jammy)
+        ```
+        echo "deb [ arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-server-6.0.gpg ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/6.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-6.0.list
+        ```
+      - Ubuntu 20.04 (Focal)
+        ```
+        echo "deb [ arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-server-6.0.gpg ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/6.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-6.0.list
```

Comment:
just call it `echo "deb [ arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-server-6.0.gpg ] https://repo.mongodb.org/apt/ubuntu {version}/mongodb-org/6.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-6.0.list` and ask users to replace version based on their ubuntu maybe? It seems unnecessary to display same command for all four versions.